vue.config.js

```diff
@@ -1,4 +1,11 @@
-const { defineConfig } = require('@vue/cli-service')
+const { defineConfig } = require("@vue/cli-service");
 module.exports = defineConfig({
-  transpileDependencies: true
-})
+  transpileDependencies: true,
+  css: {
+    loaderOptions: {
+      scss: {
+        additionalData: `@import "@/styles/global.scss";`,
```

Comment:
I will tell a bit more about it on the next Vue Dev Meeting (next Wednesday) but for now I'll give you the link and the quote 🙈 

> These files will be imported and available to every component you write, which is great for things like variables, functions, or mixins, but you should avoid any actual CSS rules. Adding CSS rules to your shared Sass files will import those rules into every component and bloat your project. For global CSS rules, create a separate file and import it into your main App.vue file instead.
source: https://austingil.com/global-sass-vue-project/